### DISCUSSION TITLE
Gitcoin的捐赠记录哪里看

### DISCUSSION THREAD

**Discussion by dahan8888** [gitcoinco/grants-stack#858] [Q&A] [answered]
Gitcoin的捐赠记录哪里看 求大神告知

**Comment by vacekj** [comment, marked as answer]
Hello @dahan8888 , the donation records for the new platform are accesible through our subgraphs. You can find documentation regarding those here: https://github.com/gitcoinco/grants-round/tree/main/packages/graph#deployed-subgraphs.
